lib/bridge-client/blacklist.js

```diff
@@ -33,30 +29,80 @@ Blacklist.TTL = 86400000;
  * Push node to blacklist
  * @param {String} nodeid - Node id to be added to blacklist
  */
-Blacklist.prototype.push = function(nodeid) {
-  this._logger.info('Adding NodeID %s to blacklist', nodeid);
-  this.blacklist[nodeid] = Date.now();
-  this._saveToDisk();
+Blacklist.prototype.push = function(nodeid, cb) {
+  var self = this;
+  self._logger.info('Adding NodeID %s to blacklist', nodeid);
+  self._getBlacklist(function() {
+    self.blacklist[nodeid] = Date.now();
+    self._saveToStore(cb);
+  });
 };
 
 /**
- * Save blacklist to disk
+ * Lazy load blacklist
  * @private
  */
-Blacklist.prototype._saveToDisk = function() {
-  fs.writeFileSync(this.blacklistFile, JSON.stringify(this.blacklist));
+Blacklist.prototype._getBlacklist = function(cb) {
+  var self = this;
+
+  // If we already have a blacklist, return it
+  if(self.blacklist !== undefined) {
```

Comment:
Keep consistent with style conventions - always add space between `if` and `()`.